docs/shared_parsers_catalog/krb5kdc_log.rst

```diff
@@ -0,0 +1,9 @@
+Kerberos KDC Logs
+=================
+
+KerberosKDCLog - ``/var/log/krb5kdc.log``
+-----------------------------------------
+.. autoclass:: insights.parsers.krb5kdc_log.KerberosKDCLog
+   :members:
+   :show-inheritance:
+
```

Comment:
@PaulWay until we decide on how we want to revamp the module documentation, this should be changed back to the current documentation standard just listing the module.